src/modules/Y2ModuleConfig.rb

```diff
@@ -335,6 +340,33 @@ def SetDesktopIcon(file)
       true
     end
 
+    # Returns list of all profile sections from the current profile, including
+    # unsupported ones, that do not have any handler (AutoYaST client) assigned
+    # at the current system and are not handled by AutoYaST itself.
+    #
+    # @return [Array<String>] of unknown profile sections
+    def unhandled_profile_sections
+      profile_sections = Profile.current.keys
+
+      profile_handlers = @ModuleMap.map{
+        |name, desc|
+        desc[RESOURCE_NAME_KEY] || name
+      }
+
+      profile_sections.reject{
+        |section|
+        profile_handlers.include?(section)
+      } - Yast::ProfileClass::GENERIC_PROFILE_SECTIONS
```

Comment:
Yes. It's more readable for me and we stick to our style guide (don't call methods on the result of a multi-block).

nitpicking: if you are going to use a one-liner, I'd cut down the `section` parameter name to just `s`. Otherwise, you could use a regular `do..end` if you prefer.